.mergify.yml

```diff
@@ -1,7 +1,7 @@
 pull_request_rules:
   - name: automatic merge
     conditions:
-      - label=automerge
+      - label!=manual merge
       - check-success=test
       - "#approved-reviews-by>=1"
```

Comment:
```suggestion
      - "#approved-reviews-by>=2"
```

?

Reply:
I know GitHub API will block it, but would it be good to enforce it here too?

Reply:
It's a good question!
We definitely can. I noticed there's a little advantage to keep the `1` here, which is that Mergify will start updating the PR (merge the base branch) as soon as it has 1 approved review. That keeps the PR ready to be merged as soon as the second approval is received.


I'll send a PR to do this in a more explicit way.